### DISCUSSION TITLE
直播画质不管怎么调都是糊的

### DISCUSSION THREAD

**Discussion by dou838731715** [ywmoyue/biliuwp-lite#355] [General]
不管怎么调都是糊的

**Comment by GD-Slime** [comment]
api获取的流好像出问题了，不管怎么样都是480p

**Comment by TsubakiAwa** [comment]
+1，接口加限制了
![image](https://github.com/ywmoyue/biliuwp-lite/assets/53134066/84b19b4c-8d99-4f41-bb41-7f084a29def9)
![image](https://github.com/ywmoyue/biliuwp-lite/assets/53134066/8d68c1db-ccdf-4ef0-9383-d545ac36eaf3)

**Comment by GD-Slime** [comment]
修了 #356 等合并或者自己改一下就行
